app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -453,16 +453,37 @@ function getConfigSelectors()
      * Returns the Skip Payment Method Step configuration.
      * @return string
      */
-    function isBoltOnlyPayment()
+    public function isBoltOnlyPayment()
```

Comment:
why public? is it accessed by other classes?

Reply:
@roopakv We definitely should be looking to use much more strict encapsulation in our code.  

Remember, unlike in Java when a method is declared without an access modifier, it does not go into `package` scope or `private` scope like in maybe some other languages, but in PHP, they are `public` by default.  So, the code above is just explicitly declaring what is implicitly invoked.